api/openapi-spec/v3/api__v1_openapi.json

```diff
@@ -5560,6 +5560,18 @@
               }
             ],
             "description": "RFC 3339 date and time at which the object was acknowledged by the Kubelet. This is before the Kubelet pulled the container image(s) for the pod."
+          },
+          "totalRequestedResources": {
+            "additionalProperties": {
+              "allOf": [
+                {
+                  "$ref": "#/components/schemas/io.k8s.apimachinery.pkg.api.resource.Quantity"
+                }
+              ],
+              "default": {}
+            },
+            "description": "TotalRequestedResources is the effective resource requests for this pod taking into consideration init and regular containers, and the pod overhead. If a resize of pod resources has been requested for a running pod, this field reflects the total desired resources for that pod which differs from actual resources allocated for the pod by the node.  If there is no effective resource request due to zero pod overhead and no requests on init or regular containers, the field is omitted. More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/",
```

Comment:
It's weird to optionally not have this set.  I would probably prefer to always set this, and have it be empty of resources.  Or maybe that's how it works, and the doc is incorrect?